crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -205,10 +206,14 @@ impl ParsedQObject {
                 ImplItem::Method(method) => {
                     self.parse_impl_method(method)?;
                 }
+                ImplItem::Const(constant) => {
+                    self.passthrough_impl_items
+                        .push(ImplItem::Const(constant.clone()));
```

Comment:
assume you can't do `item.clone()` here ?

Reply:
I can, actually :D